LiveController_deployment.md

```diff
@@ -0,0 +1,91 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - Centos 6.8+
```

Comment:
@jessp01 . actually both in production and QA the CentOS version is from 6.4 to 6.8..